src/test/run-rbd-tests

```diff
@@ -18,6 +18,7 @@ recreate_pool() {
     PG_NUM=100
     ceph osd pool delete $POOL_NAME $POOL_NAME --yes-i-really-really-mean-it
     ceph osd pool create $POOL_NAME $PG_NUM
+    rbd pool init $POOL_NAME
```

Comment:
Some plays in testing the rbd-ggate code.
But have not yet gotten around to submitting a PR.